apis/features/v1/features_types.go

```diff
@@ -29,6 +29,8 @@ const (
 	ConditionPhaseProcessTemplates = "ProcessTemplates"
 	ConditionPhaseApplyManifests   = "ApplyManifests"
 	ConditionPhasePostConditions   = "FeaturePostConditions"
+	ComponentType                  = "Component"
+	DSCIType                       = "DSCI"
```

Comment:
How about improving this slightly and make sub-type for string?

```
type OwnerType string

const (
	ComponentOwner OwnerType = "Component"
	DSCIOwner OwnerType = "DSCI"
)
```

and use this type instead? That would make it a bit more focused and type-safe. We already use something similar across the codebase - `ManagementState`

Reply:
On that note - maybe a follow-up PR to our "feature" branch could similarly make the condition phase its own type. Somehow we missed that opportunity in the original PR.

Reply:
done this here [103593f](https://github.com/maistra/opendatahub-operator/pull/17/commits/103593f01d204ded9ae413064bddcd5b8e9a5131)